app/processors/annotation_list_creator.rb

```diff
@@ -4,23 +4,26 @@ class AnnotationListCreator
 
   def initialize(identifier)
     @identifier = identifier
+    @granularity_list = %w[word line paragraph]
   end
 
-  def annotation_list
-    hr = HocrOpenAnnotationCreator.new final_hocr_filepath(@identifier)
-    if @annotation_list
-      @annotation_list
-    else
-      @annotation_list = hr.annotation_list
-    end
-  end
+ def create_annotation_lists
+    @annotation_list = Hash.new("annotation_list")
+    @granularity_list.map do |granularity|
+       hr = HocrOpenAnnotationCreator.new final_hocr_filepath(@identifier), granularity
+       hr.annotation_list
```

Comment:
Sorry. Here it is in the pull request. No need to call `hr.annotation_list` here since it is done on the next line.

Reply:
Right - thanks for catching that!